### DISCUSSION TITLE
How to add another livewire package directive?

### DISCUSSION THREAD

**Discussion by diazsasak** [filamentphp/filament#2106] [Help] [answered]
Hi, 

Does anyone know how to add livewire directive to the layout? https://github.com/wire-elements/modal#livewire-directive

thank you for your attention

**Comment by danharrin** [comment, marked as answer]
https://filamentphp.com/docs/2.x/admin/appearance#render-hooks

**Reply by diazsasak** [reply]
wow, that is great.

I never saw that in the docs before. New feature or did I miss it? 
the docs also use wire-element/modal as an example exactly the same as what I ask. Thank you very much @danharrin !

**Reply by diazsasak** [reply]
i see. new feature https://github.com/laravel-filament/filament/releases/tag/v2.11.0 :)
